aliases.py

```diff
@@ -107,4 +107,38 @@ def abilityAliases(name):
 		name='press'
 	elif name in ['mcs']:
 		name='merciless'
-	return name
\ No newline at end of file
+	elif name in ['wotw']:
+		name='way of the wind'
+	elif name in ['woi']

```

Comment:
You need colons at the end of each of these lines that start with elif

Reply:
no u